styles/styles.css

```diff
@@ -56,19 +66,20 @@ body {
   counter-reset: h4counter;
 }
 
-h2, h3 {
+h2,
+h3 {
   counter-reset: h4counter;
 }
 
 h4:before {
-  content: counter(h4counter) ".\0000a0\0000a0";
+  content: counter(h4counter) '.\0000a0\0000a0';
   counter-increment: h4counter;
   counter-reset: h3counter;
 }
 
 code {
   background-color: var(--color-gray-200);
-  padding: 0 .25rem;
+  padding: 0 0.25rem;
```

Comment:
I'd adjust the linting rules and/or prettify config you have. adding `0`s will only add bytes that aren't needed